optuna_dashboard/ts/searchSpace.ts

```diff
@@ -17,9 +17,12 @@ export const mergeUnionSearchSpace = (
       // CategoricalDistribution.choices will never be changed
       return
     }
-    d.low = Math.min(d.low, s.distribution.low)
-    d.high = Math.max(d.low, s.distribution.high)
-    knownElements.set(s.name, d)
+    const updated: Distribution = {
+      ...d,
+      low: Math.min(d.low, s.distribution.low),
+      high: Math.max(d.high, s.distribution.high),
+    }
+    knownElements.set(s.name, updated)
```

Comment:
Need to fix like this because analysis page outputs the following errors and get crashed at `d.low = Math.min(d.low, s.distribution.low)`.

```
Uncaught TypeError: Cannot assign to read only property 'low' of object '#<Object>'
```